include/pybind11/numpy.h

```diff
@@ -263,7 +282,7 @@ struct npy_api {
     };
 
     static npy_api lookup() {
-        module_ m = module_::import("numpy.core.multiarray");
+        module_ m = detail::import_numpy_core_submodule("multiarray");
         auto c = m.attr("_ARRAY_API");
         void **api_ptr = (void **) PyCapsule_GetPointer(c.ptr(), nullptr);
```

Comment:
It's an existing problem, but could you please add this fix or similar while you're at it (untested)?

```
if (api_ptr == nullptr) {
    raise_from(PyExc_SystemError, "FAILURE obtaining numpy _ARRAY_API pointer.");
}
```

Reply:
Sure, I added this `if` statement - looks good to me.